layout/js/gages.js

```diff
@@ -110,4 +110,26 @@ var hovertext;
       mapSVG.hideTooltip();
     }
   }
+  
+  vizlab.showline = function(year) {
```

Comment:
I think most of the code in this function is from the fanciness that needed to happen to filter which points are rendered and rebuild the path accordingly to truncate the path to only display sites in a given year. 

I don't know the fancy jquery way to do this, but here is a one way:
```js
vizlab.showline = function(year) {
   var lineId = '#l' + year;
   document.getElementById(lineId).setAttribute(class, 'selected-year');
}
vizlab.greyline = function(year) {
   var lineId = '#l' + year;
   document.getElementById(lineId).setAttribute(class, 'not-sure-what-class');
}
```